docs/sampling/README.md

```diff
@@ -0,0 +1,242 @@
+Records of telemetry, before sampling, in a probabilistic sense.
+
+These attributes MAY be modified by components in a collection
+pipeline to convey successive sampling that has been carried out for a
+particular item of telemetry, using the conventions for consistent
+sampling described here.  In that sense, producers and consumers
+should see these attributes as telemetry metadata.
+
+The attributes described here use non-standard naming conventions.
+Sampling relies on an ecosystem, and these specifications incorporate
+features of W3C Trace Context and from the OpenTracing project, in
+place before OpenTelemetry.  Therefore, these attributes use
+`sampling` as the prefix, instead of an OpenTelemetry-scope (e.g.,
+`otel.sampling`).
+
+## Probability sampling
+
+The OpenTelemetry sampling decision is defined in terms of a Threshold
+value and a Randomness value, each containing 56 bits of information.
+
+A constant known as _maximum adjusted count_ (`MaxAdjustedCount`),
+with value `0x100000000000000`, (which can also be expressed
+`0x1p+56`, `math.Pow(2, 56)`, or `math.Ldexp(1, 56)`), defines the
+exclusive upper limit of these values.
+
+Logically, both Threshold (`T`) and Randomness (`R`) are represented
+as unsigned integers in the range `0` through `0xffffffffffffff` or
+`MaxAdjustedCount - 1`.  Items of telemetry are selected (i.e.,
+"sampled") when their Threshold value is less than or equal to
+Randomness value, or `T <= R`.
+
+Sampling probability is defined by the following expression:
+
+```
+Probability = (MaxAdjustedCount - Threshold) / MaxAdjustedCount
+```
+
+In a Span-to-Metrics or Logs-to-Metrics pipeline, each item of
+telemetry is representative of an _adjusted count_ number of items in
+the original population.  Adjusted count is the inverse of sampling
+probability, and `MaxAdjustedCount` (defined above) is the inverse of
+the smallest supported sampling probability (which can also be
+represented as `0x1p-56`, `math.Pow(2, -56)`, or `math.Ldexp(1,
+-56)`).
+
+For the tracing signal, Threshold and Randomness propagate via W3C
+Trace Context `tracestate`.  When they appear in the `tracestate`, the
+Threshold and Randomness properties are called "T-value" and
+"R-value"; they are represented in the OpenTelemetry section of the
+`tracestate` (having vendor tag `ot`), using properties named `th` and
+`rv`, respectively.
+
+For the logs signal, which generally does not record the W3C Trace
+Context `tracestate`, sampling attributes are meant to be expressed
+using log record attributes with the same definition as T-value and
+R-value.
+
+For more information on how to perform and interpret probability
+sampling based on these properties, [consult OTEP 235][OTEP235].
+
+### Overriding sampling decisions
+
+Instrumentation authors and end-users that wish to prioritize an item
+of telemetry for collection in spite of sampling can add a
+`sampling.priority` attribute.  This attribute is inherited as a
```

Comment:
Are we adding it  for the sake of compatibility with OpenTracing? It does not seem so and I'd prefer to avoid "inhering as legacy" part and removing references to the historical context.

Reply:
Removed & done. Thanks.